packages/storybook/config/storybook/jetPieThemeDark.js

```diff
@@ -1,39 +1,39 @@
-import { create } from '@storybook/theming/create';
+import { create, color, typography } from '@storybook/theming';
 import logo from '../../public/logo--pie.svg';
 
 export default create({
     base: 'dark',
 
-    colorPrimary: '#fff',
-    colorSecondary: '#2a3846',
+    // Storybook-specific color palette
+    colorPrimary: '#f36d00',
+    colorSecondary: '#4996FD',
 
     // UI
-    appBg: '#0c151d',
-    appContentBg: '#fff',
-    appBorderColor: '#c5ccd3',
-    // appBorderRadius: 0
+    appBg: '#162028',
+    appContentBg: '#1b252e',
+    appBorderColor: 'rgba(255,255,255,.1)',
+    appBorderRadius: 4,
 
-    // Typography
+    // Fonts
     fontBase: '"JustEatBasis", Arial, sans-serif',
-    // fontCode: 'monospace',
+    fontCode: typography.fonts.mono,
 
     // Text colors
-    textColor: '#0c151d',
-    textInverseColor: 'hotpink',
+    textColor: '#ffffff',
+    textInverseColor: '#2a3846',
+    textMutedColor: color.mediumdark,
 
     // Toolbar default and active colors
-    barTextColor: '#fff',
+    barTextColor: '#ffffff',
     barSelectedColor: '#f36d00',
     barBg: '#1b252e',
 
     // Form colors
-    inputBg: '#fff',
-    inputBorder: '#0c151d',
-    inputTextColor: '#0c151d',
+    inputBg: '#111b22',
+    inputBorder: '#1b252e',
+    inputTextColor: '#f1f2f4',
     inputBorderRadius: 2,
 
-    // Meta
+    brandImage: logo,
     brandTitle: 'Just Eat Takeaway.com',
-    // brandUrl: '',
```

Comment:
If this is still relevant to the updated version of storybook, it might be worth leaving this in. I believe it's a clue to what config keys are available, as it's not obvious from the storybook documentation.